Ui/Http/Pages/Index.razor

```diff
@@ -1,44 +1,102 @@
+                    </MovieFiles.Ui.Http.Shared.Activities.ActivityList>
+                </div>
+            }
+        </Authorized>
+    </AuthorizeView>
 
-    <NotAuthorized>
-        <div class="body">
-            <FeaturedMovies MovieGetter=@featuredMoviesGetter()></FeaturedMovies>
+    @if (ActiveTab == Tab.discover)
+    {
+        <div class="movieHead">
+            <h1>Popular movies</h1>
+            <button id="filterButton" onclick="toggleFilter()">
+                <svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-adjustments-filled" width="100%"
+                    height="100%" viewBox="0 0 24 24" stroke-width="1.5" stroke="#2c3e50" fill="none" stroke-linecap="round"
+                    stroke-linejoin="round">
+                    <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+                    <path
+                        d="M6 3a1 1 0 0 1 .993 .883l.007 .117v3.171a3.001 3.001 0 0 1 0 5.658v7.171a1 1 0 0 1 -1.993 .117l-.007 -.117v-7.17a3.002 3.002 0 0 1 -1.995 -2.654l-.005 -.176l.005 -.176a3.002 3.002 0 0 1 1.995 -2.654v-3.17a1 1 0 0 1 1 -1z"
+                        stroke-width="0" fill="currentColor" />
+                    <path
+                        d="M12 3a1 1 0 0 1 .993 .883l.007 .117v9.171a3.001 3.001 0 0 1 0 5.658v1.171a1 1 0 0 1 -1.993 .117l-.007 -.117v-1.17a3.002 3.002 0 0 1 -1.995 -2.654l-.005 -.176l.005 -.176a3.002 3.002 0 0 1 1.995 -2.654v-9.17a1 1 0 0 1 1 -1z"
+                        stroke-width="0" fill="currentColor" />
+                    <path
+                        d="M18 3a1 1 0 0 1 .993 .883l.007 .117v.171a3.001 3.001 0 0 1 0 5.658v10.171a1 1 0 0 1 -1.993 .117l-.007 -.117v-10.17a3.002 3.002 0 0 1 -1.995 -2.654l-.005 -.176l.005 -.176a3.002 3.002 0 0 1 1.995 -2.654v-.17a1 1 0 0 1 1 -1z"
+                        stroke-width="0" fill="currentColor" />
+                </svg>
+                <p>Filter</p>
+            </button>
         </div>
-    </NotAuthorized>
-</AuthorizeView>
+
+        <div id="filter">
+            <FilterMovies />
+        </div>
+        <FeaturedMovies MovieGetter="moviesToDisplay"></FeaturedMovies>
+    }
+
+</div>
+
+<script suppress-error="BL9992">
+    function toggleFilter() {
+        var filter = document.getElementById("filter");
+        if (filter.style.display === "block") {
+            filter.style.display = "none";
+        } else {
+            filter.style.display = "block";
+        }
+    }
+</script>
 
 @code {
+    private int minYear { get; set; } = 1800;
+    private int maxYear { get; set; } = System.DateTime.Now.Year;
+    private bool showFilterBox = false;
+    private Func<int, Task<MovieList>>? moviesToDisplay = null;
+    private GenreList genreList = new() { Genres = new() };
+    private int selectedGenreId { get; set; } = emptyGenre.Id;
+    private static readonly Genre emptyGenre = new() { Id = -1, Name = "---" };
+    private Person? filterCrew { get; set; }
```

Comment:
Not sure if these should stay here as Andrej moved them to Ui/Http/Shared/Filter/FilterMovies.razor